pkg/network/ebpf/c/protocols/classification/dispatcher-helpers.h

```diff
@@ -13,24 +13,6 @@
 #include "protocols/http2/helpers.h"
 #include "protocols/kafka/kafka-classification.h"
 
-static __always_inline bool is_http_monitoring_enabled() {
-    __u64 val = 0;
-    LOAD_CONSTANT("http_monitoring_enabled", val);
```

Comment:
https://github.com/DataDog/datadog-agent/blob/main/pkg/network/protocols/http/ebpf_main.go#L400
addBoolConst() do the job

Reply:
nicolas is right
The current PR takes the changes in #16663 and generalize them (via the macro), and expand the usage

I'll add that to the PR description

Reply:
oh, I see! I tried to find it in my local `main` branch, but I guess I was looking at stale code since it was merged 2 days ago 🤦 
thanks for the clarification